README.md

```diff
@@ -1,6 +1,6 @@
-# Fleek Non Fungible Apps
+# Fleek Non-Fungible Apps

```

Comment:
I think adding `-` should be double checked with marketing side of the project before merging it

Reply:
I confirmed the hyphened version with Nico.